FirebaseVertexAI/Tests/Unit/VertexComponentTests.swift

```diff
@@ -12,25 +12,29 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import Foundation
-
 import FirebaseCore
-@testable import FirebaseVertexAI
-
+import Foundation
 import SharedTestUtilities
-
 import XCTest
 
+@testable import FirebaseVertexAI
+
 @available(iOS 15.0, macOS 11.0, macCatalyst 15.0, tvOS 15.0, watchOS 8.0, *)
 class VertexComponentTests: XCTestCase {
-  static var app: FirebaseApp!
+  static let projectID = "test-project-id"
+  static let apiKey = "test-api-key"
+
+  static var app: FirebaseApp?
+
+  let location = "test-location"
```

Comment:
Should the other constants also be plain `let` instead of `static let`?

Reply:
Unfortunately we can't in this case (even though they're constants) since we're using them in `class func setUp()` too.